catalog.html

```diff
@@ -145,22 +143,32 @@ <h2 class="visually-hidden">Сортирование</h2>
         </select>
       </form>
       <a href="#" class="button">
-        <span class="visually-hidden">Показать карточками</span>
+        <span class="visually-hidden">
+          <img src="images/catalog/mode-tile.png" alt="Показать карточками">
+        </span>
       </a>
       <a href="#" class="button">
-        <span class="visually-hidden">Показать в виде галереи</span>
+        <span class="visually-hidden">
+          <img src="images/catalog/mode-slideshow.png" alt="Показать в виде галереи">
+        </span>
       </a>
       <a href="#" class="button button-dark">
-        <span class="visually-hidden">Показать списком</span>
+        <span class="visually-hidden">
+          <img src="images/catalog/mode-list.png" alt="Показать списком">
+        </span>
       </a>
     </section>
     <section class="catalog-list-section">
       <h2 class="visually-hidden">Список гостиниц</h2>
       <div class="hotel">
         <a class="hotel-link" href="#">
-          <img class="hotel-image" src="" alt="Amara Resort & Spa">
+          <img class="hotel-image" src="images/catalog/hotel-first.jpg" alt="Amara Resort & Spa">
           <h3 class="hotel-title">Amara Resort & Spa</h3>
-          <span class="visually-hidden">Четыре звезды</span>
+          <p class="stars four-stars">
+           <span class="visually-hidden">
+             <img src="images/catalog/star.png" alt="Четыре звезды">
+           </span>
+          </p> 
```

Comment:
Вроде бы столько мучались с тем, что нужно добавить сюда информацию не картинкой и текстом, а тут опять тоже самое.
```suggestion
          <p class="stars four-stars">
            <span class="visually-hidden">Четыре звезды</span>
          </p> 
```